spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+feature 'liking posts' do
+  before do
+    testpost = Post.create(caption: 'Here is a test post')
+  end
+
+  scenario 'a user can like a post, which increments the post like counter', js: true do
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)